db/migrate/20151021014315_create_course_assessment_question_text_response.rb

```diff
@@ -0,0 +1,25 @@
+class CreateCourseAssessmentQuestionTextResponse < ActiveRecord::Migration
+  def change
+    create_table :course_assessment_question_text_responses do
+    end
+
+    create_table :course_assessment_question_text_response_solutions do |t|
+      t.references :question,
+                   null: false,
+                   index: {
+                     name: :fk__course_assessment_text_response_solution_question
```

Comment:
same as above, was following https://github.com/Coursemology/coursemology2/blob/master/db/migrate/20150726062900_create_course_assessment_question_multiple_response.rb so not too sure :/